Userland/allocate.cpp

```diff
@@ -8,7 +8,9 @@ void usage(void)
     exit(1);
 }
 
-enum Unit { Bytes, KiloBytes, MegaBytes };
+enum Unit { Bytes,
+    KiloBytes,
+    MegaBytes };
```

Comment:
...or this.
Okay let's just not run clang-format on your machine until you have a newer version ;)

Reply:
Whoops! I'll upgrade and try again :D